elyra/validation/handlers.py

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright 2018-2021 Elyra Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from elyra.validation import PipelineValidationManager
+from elyra.util.path import get_expanded_path
+from jupyter_server.base.handlers import APIHandler
+from tornado import web
+from ..util.http import HttpErrorMixin
+
+
+class PipelineValidationHandler(HttpErrorMixin, APIHandler):
```

Comment:
Same "location" comment, should this placed into pipeline/handlers.py?

Reply:
updated